packages/main/src/internal/withWebComponent.tsx

```diff
@@ -52,8 +52,14 @@ export const withWebComponent = <T extends any>(
     return Object.keys(getWebComponentMetadata().getSlots());
   };
 
-  const getEventsFromMetadata = (): string[] => {
-    return Object.keys(getWebComponentMetadata().getEvents() || {}).filter((eventName) => !eventName.startsWith('_'));
+  const getEventsFromMetadata = (otherProps?: object): string[] => {
```

Comment:
```suggestion
  const getEventsFromMetadata = (otherProps: = {}): string[] => {
```
You can use the default parameter value here, than you can skip the manual check in line 59